docs/BR.md

```diff
@@ -1286,29 +1286,36 @@ No stipulation.
 
 ### 4.9.7 CRL issuance frequency
 
-CAs MUST generate and publish either:
-- a full and complete; OR
+CRLs must be available via a publicly-accessible HTTP URL (i.e., "published").
+
+Within 24 hours of issuing its first Certificate, CAs MUST generate and publish either:
+- a full and complete CRL; OR
 - partitioned (i.e., "sharded") CRLs, that when aggregated, represent the equivalent of the full and complete CRL.
 
-CRLs must be available via a publicly-accessible HTTP URL.
+CAs issuing Subscriber Certififcates MUST continue to issue CRLs in accordance with the table below:
+
+| __Conditions__ | __Minimum CRL Issuance Frequency__ | __Maximum nextUpdate Period__ | 
+| ----           | -                                  | -                             |
+| The CA has revoked at least one unexpired Certificate since the `thisUpdate` field of the last CRL. | A CRL MUST be generated and published within 24 hours of the `thisUpdate` field of the last CRL. <br><br> A CRL SHOULD be generated and published within 4 hours of the `thisUpdate` field of the current CRL. | The value of the `nextUpdate` field MUST NOT be more than 10 days beyond the value of the `thisUpdate` field. |
```

Comment:
> I must admit I'm still not enamored with this use of "has revoked". Despite Dimitris' totally valid point that we already do this for Subordinate CA Certificates, to me it feels like an unfortunate precedent to carry forward. How can a CA be said to "have revoked" something if they haven't published a CRL containing it yet?
> 
> Also, this phrasing is impossible. Suppose that a CA issues a CRL, and then coasts for 5 days with no revocations being necessary. This is permitted under the "All other conditions" entry. Then, a cert needs to be revoked. But significantly more than 24 hours have already passed since the `thisUpdate` of the previous CRL! So in practice, a CA would be required to issue at least every 24 hours, just in case a new revocation request comes in.
> 
> These kinds of accidental gotchas are why I'm opposed to a two-tiered system of CRL issuance frequency requirements. We've already demonstrated that it is difficult to write and read (I missed this contradiction on my first two reads of this diff!) requirements like this. We should not be making things more difficult than necessary.
> 
> I'm still in favor of the previous "CAs issuing Subscriber Certificates SHALL update and reissue CRLs at least once every 24 hours".

I believe this reading is a bit egregious. Every CA must record the status of a certificate (intermediate or end-entity). Due to the way PKI works, there is always some delay (greater than zero), between changing the status of a certificate to "revoked" and issuing a CRL or signing a new OCSP response. This is especially obvious when CAs perform offline ceremonies with CRLs/OCSP responses issued from Root CAs. I hope people agree with this.

I [proposed some text](https://github.com/ryancdickson/staging/pull/3#discussion_r1186875190) that I consider clear and unambiguous. I hope more Members can chime-in and state their opinion whether this language works and delivers what is expected.

Reply:
(Reviewed and incorporated @dzacharo suggested edits.)